test/spec/exporters/jhipster_application_exporter.spec.js

```diff
@@ -181,7 +205,7 @@ describe('JHipsterApplicationExporter', () => {
           fs.unlinkSync(path.join('.yo-rc.json'));
         });
 
-        it('adds the read content to the exported application', () => {
+        it('adds the read content to the exported application and keep selected', () => {
```

Comment:
2 things:
  - what does keep selected mean?
  - if there's an "and" in an "it", it usually means you're testing two things and this should be split in two "it"